index.css

```diff
@@ -165,18 +165,19 @@ input:checked ~ nav {
 
 nav ul {
   position: relative;
-  top: 30px;
+  top: 90px;
   font-size: clamp(18px, 6vw, 36px);
   text-align: center;
 }
 
 .nav-item {
-  margin: 6vw;
+  margin: 3vw;
 }
 
 .nav-link {
   text-decoration: none;
   color: var(--main);
+  margin: 6vw;
 }
```

Comment:
- [ ] Good job on making some changes requested by the previous review :+1:  but there still something to improve for your project to look professional , as required for this project All page elements have similar spacing, borders, shadows, and paddings to the original design. kindly fix this so that your project will match the original , your design missed something if we compare it to the original one :wink: 
Your Design 
![bro](https://user-images.githubusercontent.com/88894525/218199577-7f134c48-73ae-46e4-aa55-0cf589dbbaf4.png)


Original Design:
![original](https://user-images.githubusercontent.com/88894525/218199604-e348dc31-752e-4b4a-8854-848cce80f20e.png)

Reply:
Hi @Alfredbis29 ! Thank you for your comments!
About the spacing in the headline, between the `<nav>` and the **title**, you can see that the "HELLO! SHARING WORLD!" text is committed and the "EL FESTIVAL INDOMABLE" slogan is taking that space. This way we are taking advantage of the background image, and the purpose of showing the context is fulfilled; and, based on the rest of the page, you can see that all margins and paddings are achieved as desired, showing there is no lack of skill or error here.